packages/blade/src/components/Carousel/Carousel.stories.tsx

```diff
@@ -436,6 +438,70 @@ AutoPlay.argTypes = {
   },
 };
 
+export const Uncontrolled: StoryFn<typeof CarouselComponent> = (props) => {
+  return (
+    <Box margin="auto" padding="spacing.4" width="100%">
+      <Text marginY="spacing.5">
+        Setting `defaultActiveSlide` you can provide the initial active slide and use the carousel
+        in an uncontrolled way.
+      </Text>
+      <CarouselExample
+        {...props}
+        defaultActiveSlide={2}
+        onChange={(slideIndex) => {
+          console.log('slideIndex', slideIndex);
+        }}
+      />
+    </Box>
+  );
+};
+
+Uncontrolled.args = {
+  visibleItems: 2,
+};
+Uncontrolled.argTypes = {
+  shouldAddStartEndSpacing: {
+    table: {
+      disable: true,
+    },
+  },
+};
+
+export const Controlled: StoryFn<typeof CarouselComponent> = (props) => {
+  const [activeSlide, setActiveSlide] = React.useState(0);
+
+  return (
+    <Box margin="auto" padding="spacing.4" width="100%">
+      <Text marginY="spacing.5">
+        Setting <Code>activeSlide</Code> & <Code>onChange</Code> you can control the active slide
+        and use the carousel in a controlled way. Here the active slide is {activeSlide}
+      </Text>
+      <Button marginY="spacing.4" size="small" onClick={() => setActiveSlide(2)}>
+        Go to slide #3
+      </Button>
+      <CarouselExample
+        {...props}
+        activeSlide={activeSlide}
```

Comment:
Should we still call this prop `value`? E.g. in select input, we set selected items using prop `value`.

`activeSlide` + `onChange` combination feels a little strange as opposed to `activeSlide` + `onActiveSlideChange`, `value` + `onChange`

Reply:
Sure it sounds a bit strange because we initially called the onChange as `onChange` instead of something else like `onSlideChange` 

But value is also strange in context of carousel, in input it makes sense because input's have "values" but in carousel there no "value" as such, so activeSlide may be better in this case.

Reply:
hmm yeah its not input so makes sense I think